src/main/java/io/confluent/idesidecar/restapi/resources/RestProxyResource.java

```diff
@@ -47,18 +46,24 @@ public class RestProxyResource {
   static final MultiMap NO_HEADERS = MultiMap.caseInsensitiveMultiMap();
   static final Map<String, String> NO_PATH_PARAMS = Map.of();
 
-  // Add this static declaration alongside the other static constants
-  static final String ccloudApiProxyRegex = ConfigProvider
+  static final String ccloudApiControlPlaneProxyRegex = ConfigProvider
       .getConfig()
       .getValue("ide-sidecar.proxy.ccloud-api-control-plane-regex", String.class);
 
+  static final String ccloudApiDataPlaneProxyRegex = ConfigProvider
+      .getConfig()
+      .getValue("ide-sidecar.proxy.ccloud-api-data-plane-regex", String.class);
+
   @Inject
   Router router;
 
   void init(@Observes StartupEvent ev) {
-    router.routeWithRegex(ccloudApiProxyRegex)
+    router.routeWithRegex(ccloudApiControlPlaneProxyRegex)
         .handler(BodyHandler.create())
-        .blockingHandler(this::ccloudProxy);
+        .blockingHandler(this::ccloudControlPlaneProxy);
+    router.routeWithRegex(ccloudApiDataPlaneProxyRegex)
+        .handler(BodyHandler.create())
+        .blockingHandler(this::ccloudDataPlaneProxy);
```

Comment:
Did you originally add this method, so that you can load the regex from the application config?

Reply:
yes, renamed and added another one for the data plane.

```
  static final String ccloudApiControlPlaneProxyRegex = ConfigProvider
      .getConfig()
      .getValue("ide-sidecar.proxy.ccloud-api-control-plane-regex", String.class);

  static final String ccloudApiDataPlaneProxyRegex = ConfigProvider
      .getConfig()
      .getValue("ide-sidecar.proxy.ccloud-api-flink-data-plane-regex", String.class);
```

bit of a repeat, could factor out...